conf/messages

```diff
@@ -33,102 +19,71 @@ image.create.failed=Failed to create image
 auth.tokenDeleted=Token was deleted
 auth.invalidToken=The token is invalid
 auth.token.noUser=Could not determine user for access token
-auth.invite.invalidToken=The invite token is invalid or expired.
 
-team.created=Team was created
-team.notAllowed=You are not part of this team. Project can’t be created.
-team.admin.notPossibleBy=User {1} can’t be assigned administrative rights in team {0} because he doesn't belong to the teams parent team.
+team.created=Team was created successfully
+team.admin.notPossibleBy=User {1} cannot be assigned administrative rights in team {0} because he does not belong to the teams parent team.
 team.admin.notAllowed=You are not authorized to administrate this team.
 team.notFound=Team could not be found.
-team.nameInUse=Team name is or was already in use. Please chose a different name.
-team.delete.organizationTeam=Team cannot be deleted as it is the organization base team
+team.nameInUse=This name is already assigned to a different team and not available anymore. Please choose a different name.
+team.delete.organizationTeam=This team cannot be deleted. Each organization requires at least one base team.
 team.delete.inUse=Team cannot be deleted as it is referenced in an at least one annotation, project or task type
 team.inUse.annotations=Team is referenced by {0} annotations
 team.inUse.projects=Team is referenced by {0} projects
-team.inUse.tasktypes=Team is referenced by {0} task types
 
 organization.create.forbidden=You are not allowed to create a new organization
 organization.create.failed=Failed to create a new organization
-organization.notFound=Organization {0} couldn’t be found
+organization.notFound=Organization {0} could not be found
 organization.list.failed=Failed to retrieve list of organizations.
-organization.name.invalid=The chosen organization name is invalid.
-organization.name.alreadyInUse=The chosen organization name is already in use.
+organization.name.invalid=This organization name contains illegal characters. Please only use letters and numbers.
+organization.name.alreadyInUse=This name is already claimed by a different organization and not available anymore. Please choose a different name.
 organization.alreadyJoined=Your account is already associated with the selected organization.
 
-user.activated={0} has been activated
 user.notFound=User not found
 user.noAdmin=Access denied. Only admin users can execute this operation.
-user.deactivated=Your account hasn’t been activated yet
+user.deactivated=Your account has not been activated by an admin yet. Please contact your organisation's admin for help.
```

Comment:
```suggestion
user.deactivated=Your account has not been activated by an admin yet. Please contact your organization’s admin for help.
```

Reply:
We use organization with a z (american english) in wk everywhere, as far as I can tell